cli/app.go

```diff
@@ -1585,33 +1652,28 @@ var app = &cli.App{
 										&cli.StringFlag{
 											Name:     trainFlagModelOrgID,
 											Required: true,
-											Usage:    "organization ID to upload and run training job",
+											Usage:    "org ID to upload and run training job",
 										},
 										&cli.StringFlag{
 											Name:     mlTrainingFlagName,
 											Usage:    "script name of the ML training script to upload",
 											Required: true,
 										},
 										&cli.StringFlag{
-											Name:     mlTrainingFlagVersion,
-											Usage:    "version of the ML training script to upload. defaults to current timestamp if unspecified.",
-											Required: false,
+											Name:  generalFlagVersion,
+											Usage: "version of the ML training script to upload. defaults to current timestamp if unspecified.",
```

Comment:
(minor) should we get rid of the "defaults to" sentence here and add a `DefaultText: current timestamp`?